regression/cbmc-java/virtual9/test.desc

```diff
@@ -0,0 +1,13 @@
+KNOWNBUG
+Test.class
+--program-only
+^EXIT=0$
+^SIGNAL=0$
+B.f
+C.f
+--
+A.f
+D.f
+E.f
+--
+could be pruned by propagating clsids in goto-symex
```

Comment:
I'm afraid that's not sufficient, you'd have to constant-propagate a lot more, because the guard `args.length==3` introduces a non-constant expression. (The class identifiers would be sorted out by #917.)

Reply:
Propagating constants is not enough in this case, it would require propagating a (bounded) value set.